content/en/docs/admin/admission-controllers.md

```diff
@@ -532,21 +517,78 @@ for more information.
 
 ### PodTolerationRestriction
 
-This admission controller first verifies any conflict between a pod's tolerations and its
-namespace's tolerations, and rejects the pod request if there is a conflict.
-It then merges the namespace's tolerations into the pod's tolerations.
-The resulting tolerations are checked against the namespace's whitelist of
-tolerations. If the check succeeds, the pod request is admitted otherwise
-rejected.
+This admission controller defaults and limits what tolerations may be used on Pods in a given namespace.
+The plugin will build a list of defaults, and a whitelist of allows tolerations based on annotations on the namespace.
+When the namespace does not specify defaults, the cluster defaults from the plugin configuration are used.
+When the namespace does not specify a whitelist, the namespace-specifc whitelist from the plugin configuration are used.
+
+**Note:** If the annotations are specified, but empty, they are considered to override the cluster-level defaults.
+{: .note}
+
+#### Configuration File Format
+
+PodTolerationRestriction uses a configuration file to define:
+
+1. A cluster-wide default node selector (this is used if the `` annotation is not present on the target Namespace)
 
-If the pod's namespace does not have any associated default or whitelist of
-tolerations, then the cluster-level default or whitelist of tolerations are used
-instead if specified.
+2. Namespace-specific default whitelists (these are used if the `` annotation is not present on the target Namespace)
 
-Tolerations to a namespace are assigned via the
-`scheduler.alpha.kubernetes.io/defaultTolerations` and
-`scheduler.alpha.kubernetes.io/tolerationsWhitelist`
-annotation keys.
+Note that the configuration file format will move to a versioned file in a future release.
+This file may be json or yaml and has the following format:
+
+```yaml
+apiVersion: podtolerationrestriction.admission.k8s.io/v1alpha1
+      kind: Configuration
+      default:
+        - key: key1
+          operator: Equal
+          value: value1
+      whitelist:
+        - key: key1
+          operator: Equal
+          value: value1
+        - key: key2
+          value: value2
+```
+
+Reference the PodNodeSelector configuration file from the file provided to the API server's command line flag `--admission-control-config-file`:
+
+```yaml
+kind: AdmissionConfiguration
+apiVersion: apiserver.k8s.io/v1alpha1
+plugins:
+- name: PodTolerationRestriction
+  path: podtolerationrestriction.yaml
+...
```

Comment:
Is it possible to include a complete and usable YAML example instead of the ellipses?